blog/content/post/2019-06-03-Hummingboard-Pulse-first-impression.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Hummingboard Pulse - first impression.
+abstract: In this post, we will take a look at one of the SolidRun product -
+          the HummingBoard Pulse. After power up the board we will try to boot
+          operating system on it.
+cover: /covers/hummboard.jpg
+author: tomasz.zyjewski
+layout: post
+published: false
+date: 2019-06-10
+archives: "2019"
+
+tags:
+  - HummingboardPulse
+  - SOM
+  - i.MX8
+  - u-boot
+  - linux
+categories:
+  - Firmware
+
+---
+
+### Introduction
+
+SolidRun is a global leading developer of embedded systems and he provides
+a lot of powerful products. One of them I want to describe in this post and
```

Comment:
`One of them I want to describe in this post is the HummingBoard Pulse.` should be enough (`and it ` removed)